motion/p5js/cones/index.js

```diff
@@ -0,0 +1,72 @@
+const height = 60;
+const radius = 60;
+let y = 0;
+let increment = 1;
+
+function setup() {
+  createCanvas(500, 500, WEBGL);
```

Comment:
The canvas needs to be assigned to the sketch div. See the documentation on contributing p5js sketches https://github.com/fabianmoronzirfas/gestalten-in-code/wiki/Contributing-P5js-Sketches#code